projects/core/src/error-handling/cx-error-handler.ts

```diff
@@ -4,14 +4,56 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { ErrorHandler, Injectable, inject } from '@angular/core';
-import { LoggerService } from '../logger/logger.service';
+import { ErrorHandler, Injectable, Injector, inject } from '@angular/core';
+import { FeatureConfigService } from '../features-config';
+import { LoggerService } from '../logger';
+import {
+  ERROR_INTERCEPTORS,
+  ErrorInterceptor,
+} from './error-interceptors/error-interceptor';
+import {
+  handleInterceptors,
+  sortErrorInterceptors,
+  tailChain,
+} from './utils/error-interceptor-utils';
 
+/**
+ * The CxErrorHandler is the default ErrorHandler for Spartacus in SSR mode.
```

Comment:
[MEDIUM]
It's always a default ErrorHandler in Spartacus.
```suggestion
 * The CxErrorHandler is the default ErrorHandler for Spartacus.
```